### PR TITLE
Microsoft Partner Center (Independent Publisher)

### PR DESCRIPTION
**Pull request by oleksii-innoware**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by oleksii-innoware** [comment]
Get All Customers in flow
Get customer details in flow
Get invoices in flow

**Comment by oleksii-innoware** [comment]
GetInvoices connector
GetAllCustomers in connector
Get Customer Subscriptions connector
GetCustomerOrders connector
GetCustomerDetails in connector

**Comment by v-EgorKozhadei** [comment]
Hello @oleksii-innoware, 

Thank you very much and congratulations on your new submission. 

Please let us know if you're still working on connector files.

If you have finished your work, you can remove the word "Proposal" from Pull Request's title.
After you change the title, we will remove the "Proposal" label and our Certification Engineers will review the PR within 1-2 weeks.

It's required to have three files in a PR: readme.md, apiDefinition.swagger.json and apiProperties.json.
Please rename Microsoft-Partner-Center.swagger.json file to [apiDefinition.swagger.json](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json).

If you need any additional information, you can refer to the [documentation for independent publishers](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission-ip), [connector coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards), [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance) and [templates for independent publisher connector files](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/Independent%20Publisher).

Please let us know if you have any questions. 

Thank you very much again for working with us.

**Comment by oleksii-innoware** [comment]
Hi @v-EgorKozhadei 
Thank you for your feedback. I fixed the issues you mentioned.

**Comment by v-EgorKozhadei** [comment]
Hello @oleksii-innoware,

Thank you for an update! Removing "Proposal" label.
Our Certification Engineers will review the PR within 1-2 weeks.

Let us know if you have any questions.

File: independent-publisher-connectors/Microsoft Partner Center/apiDefinition.swagger.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft Partner Center/apiDefinition.swagger.json:484]: @oleksii-innoware All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — DISMISSED** [review]

File: independent-publisher-connectors/Microsoft Partner Center/apiDefinition.swagger.json
Comment (oleksii-innoware) [reply independent-publisher-connectors/Microsoft Partner Center/apiDefinition.swagger.json]: Hi @sriyen-msft 
Thank you for feedback - I fixed some descriptions. Almost all descriptions I took from MS Docs documentation, but if you find issues - kindly point me as I am not a native speaker and can make mistakes.
Thank you in advance and have a nice day!

**Comment by sriyen-msft** [comment]
@oleksii-innoware We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @oleksii-innoware,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by oleksii-innoware** [comment]
> Hello @oleksii-innoware, I hope you are doing well. Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Please let us know if you have any questions. Thank you very much for working with us.

Hi team.
Kindly update the status as I still don't have any feedback on deployment status. Thank you in advance.

**Comment by oleksii-innoware** [comment]
Hi team,
Kindly update as I could see my PR is still is not merged.
Thank you

Oleksii Skirko

Systems Engineer

Innoware |  tel.: +380 44 4902220 | mob.: +380 97 3880395
***@***.***
Microsoft Gold ERP| Cloud Productivity| Cloud Platform Partner
6th floor, 3 Zolotoustivska str. | Kyiv 01135 | Ukraine |  www.innoware.com<http://www.innoware.com/>
***@***.***<https://www.facebook.com/InnowareHQ/>  ***@***.*** <https://www.linkedin.com/company/innoware>   ***@***.*** <https://www.youtube.com/channel/UCQ4yhOWttvdCX0s13uXBE1g>


***@***.***

**Comment by Amjed-Ayoub** [comment]
Hi @oleksii-innoware apologies for the delay in the progress.

Our team is still working on getting the proper permissions for your connector, please expect to hear back from us within 2 weeks.

Thank you so much again for your patience and understanding.

**Comment by Amjed-Ayoub** [comment]
Hello @oleksii-innoware,

I hope you are doing well.

This to inform you that we have applied for a 1st party app for your connector, and we are waiting for a response. 
We will keep you following up with the appropriate team but this usually takes about a month. we submitted 2 weeks ago. hopefully we can hear back in January.
I will keep you posted.

Thank you so much again for your patience and understanding.

**Comment by oleksii-innoware** [comment]
Hi team,
Kindly update on this PR.
Thank you in advance

**Comment by Amjed-Ayoub** [comment]
Hello @oleksii-innoware,

Thank you so much for following up with us.

We have got the approval for the 1st party app. we will prepare an Icon for the connector and proceed with the deployment process. Please expect this to complete within 3-4 weeks.

Thank you so much again for your patience and understanding.

**Comment by Amjed-Ayoub** [comment]
Hello @oleksii-innoware ,
I hope you are doing well. please accept my apologies for the delay in progress.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @oleksii-innoware ,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Comment by oleksii-innoware** [comment]
Hi Amjed.
Thank you for good news. I appreciate your support and collaboration.

**Comment by oleksii-innoware** [comment]
Hi Amjed,
Hope you are doing well. I tried to create some flow with this connector, but got an error. Also I was asked a Microsoft support guy regarding this issue.
***@***.***
Could you please clarify why we get this error and how to avoid it?
Many thanks in advance.

Oleksii Skirko

Systems Engineer

Innoware |  tel.: +380 44 4902220 | mob.: +380 97 3880395
***@***.***
Microsoft Gold ERP| Cloud Productivity| Cloud Platform Partner
6th floor, 3 Zolotoustivska str. | Kyiv 01135 | Ukraine |  www.innoware.com<http://www.innoware.com/>
***@***.***<https://www.facebook.com/InnowareHQ/>  ***@***.*** <https://www.linkedin.com/company/innoware>   ***@***.*** <https://www.youtube.com/channel/UCQ4yhOWttvdCX0s13uXBE1g>


***@***.***

**Comment by Amjed-Ayoub** [comment]
@oleksii-innoware may I you please start an email thread with our Connector Partner Mgmt Team <ConnectorPartnerMgmtTeam@service.microsoft.com> on this?
Thank you so much.

**Comment by oleksii-innoware** [comment]
Hi Amjed,
Yes sure, please proceed. Also I always ready to help or make required changes to my connector in case of need.
Thank you.

Oleksii Skirko

Systems Engineer

Innoware |  tel.: +380 44 4902220 | mob.: +380 97 3880395
***@***.***
Microsoft Gold ERP| Cloud Productivity| Cloud Platform Partner
6th floor, 3 Zolotoustivska str. | Kyiv 01135 | Ukraine |  www.innoware.com<http://www.innoware.com/>
***@***.***<https://www.facebook.com/InnowareHQ/>  ***@***.*** <https://www.linkedin.com/company/innoware>   ***@***.*** <https://www.youtube.com/channel/UCQ4yhOWttvdCX0s13uXBE1g>


***@***.***

**Comment by Amjed-Ayoub** [comment]
@oleksii-innoware please send us an email :-)

**Pull request merged**
